### PR TITLE
Adding Babel dependencies to resolve regenerator/static test failures

### PR DESCRIPTION
**Pull request by stephenkao**
@dtothefp It worked!

I added new Babel plugins and the Node 4 preset to get the tests running.  Also added a new very, very simple end-to-end test to make sure our bases are covered in case of `async` and `static` regressions.

I feel like you've saved my hide one hundred times already this week, and it's only Monday.  :P

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "babel-plugin-typecheck": "^3.5.1",
     "babel-polyfill": "^6.3.14",
     "babel-preset-es2015": "^6.1.18",
+    "babel-preset-es2015-node4": "^2.1.0",
     "babel-preset-react": "^6.1.18",
     "babel-preset-stage-0": "^6.1.18",
```
Comment (dtothefp) [inline package.json]: this is in the wrong package.json, needs to be in `boiler-task-selenium`

File: packages/boiler-task-selenium/src/runner/webdriverio/index.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
 /**
  * Bootstrap the es6 process here for test files and config
  */
+const presetPath = require.resolve(process.cwd() + '/packages/boiler-task-selenium/node_modules/babel-preset-es2015-node4');
 require('babel-register')({
-  presets: ['es2015', 'stage-0'],
```
Comment (dtothefp) [inline packages/boiler-task-selenium/src/runner/webdriverio/index.js]: can just do `require.resolve`

**Comment by dtothefp** [comment]
LGTM...ugghhhh webriver :neckbeard:

**Comment by stephenkao** [comment]
:neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard: :neckbeard:

**Pull request merged**